src/SFA.DAS.AssessorService.Application/Handlers/Apply/Review/WithdrawalApplicationsHandler.cs

```diff
@@ -33,8 +37,40 @@ public async Task<PaginatedList<ApplicationSummaryItem>> Handle(WithdrawalApplic
 
             var items = Mapper.Map<IEnumerable<ApplicationListItem>, IEnumerable<ApplicationSummaryItem>>(organisationApplicationsResult.PageOfResults);
 
+            if(items.Any())
+            {
+                // only want to do this once.
+                var allEnrolledVersions = await _standardRepository.GetEpaoRegisteredStandardVersions(items.First().EndPointAssessorOrganisationId);
+
+                // Get versions for each standard and determine the withdrawal application type.
+                foreach (var item in items)
+                {
+                    var allEnrolledStandardVersions = allEnrolledVersions.Where(v => v.IFateReferenceNumber == item.StandardReference).ToList();
+                    item.WithdrawalType = GetWithdrawalApplicationType(item, allEnrolledStandardVersions);
+                }
+            }
+
+
             return new PaginatedList<ApplicationSummaryItem>(items.ToList(),
                     organisationApplicationsResult.TotalCount, request.PageIndex, request.PageSize, request.PageSetSize);
         }
+
+        // SV-912 Helper to generate the type of withdrawal
+        private string GetWithdrawalApplicationType(ApplicationSummaryItem item, IEnumerable<StandardVersion> allEnrolledVersions)
```

Comment:
Same here for tweaks
allEnrolledVersions -> enrolledVersionsForStandard?

then inside the loop
allVersions -> maybe versionNumberList?

I think the names are a hangover from the previous implementation but i think a few adjustments makes it more explicit.